### DISCUSSION TITLE
Atari 50 The Anniversary Celebration

### DISCUSSION THREAD

**Discussion by dudulins** [farmerbb/RED-Project#176] [Ideas]
Has anyone tried extracting the ROMs from the latest Atari compilation? It uses the same assets.pie package and I believe It can extracted using the same method/scripts in RED-Project if the file offsets are identified. Any ideas?

**Comment by mjBrickman** [comment]
There is a tool for that, but extracting the ROM files involves decryption, which could be interpreted as illegally circumventing DRM. https://github.com/Masquerade64/Cowabunga
